lib/assessment.rb

```diff
@@ -44,5 +49,7 @@ def transform(words)
   # space and return the result.
   def read_file(filename)
     # Your code goes here
+    filename.reduce(&:+)
```

Comment:
Nope. This one's tricky - take a look at Antony's solution branch and see if you can figure out what he's doing. Remember, from the perspective of the console, a file is just an array of lines of text.